app/controllers/admin/health/scheduled_documents_controller.rb

```diff
@@ -0,0 +1,89 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Admin::Health
+  class ScheduledDocumentsController < HealthController
+    before_action :require_has_administrative_access_to_health!
+    before_action :require_can_manage_accountable_care_organizations!
+    before_action :set_scheduled_document, only: [:edit, :update, :destroy]
+
+    ALLOWED_DOCUMENT_CLASSES = [
+      'Health::ScheduledDocuments::EnrollmentDisenrollment',
+    ].freeze
+
+    def index
+      @scheduled_documents = scheduled_document_source.all
+    end
+
+    def new
+      document_type = params[:type]
+      raise 'Invalid Document Type' unless ALLOWED_DOCUMENT_CLASSES.include?(document_type)
+
+      @scheduled_document = scheduled_document_source.new(type: document_type)
+    end
+
+    def create
+      document_type = new_scheduled_document_params[:type]
+      raise 'Invalid Document Type' unless ALLOWED_DOCUMENT_CLASSES.include?(document_type)
+
+      scheduled_document = scheduled_document_source.create(type: document_type)
+      scheduled_document.update!(scheduled_document_params(scheduled_document))
```

Comment:
Why is this two lines? 
```suggestion
      @scheduled_document = scheduled_document_source.create!({type: document_type}.merge({scheduled_document_params(scheduled_document)})
      respond_with(@scheduled_document)
```

Reply:
`scheduled_document_params` relies on being able to query the `scheduled_document` subclass to know what its type-specific attributes are, and I didn't want updates to be able to change the STI.

If the concern is the extra DB access, we could trade it for an extra alloc with `scheduled_document_source.new(type: document_type)`

Reply:
how about:
```
options = scheduled_document_params(scheduled_document).merge({type: document_type})
@scheduled_document = scheduled_document_source.create!(options)
```
I'm not opposed to the extra query, but it will probably mess with `respond_with`, which was the main change I was looking for.